lib/validators/specResolver.ts

```diff
@@ -1027,4 +1033,25 @@ export class SpecResolver {
     }
     return result
   }
+
+  /**
+   * Check if exist undefined within-document reference
+   */
+  private checkWithInReferences() {
+    const errsDetail: any[] = []
+    const unresolvedRefs = jsonUtils.findUndefinedWithinDocRefs(this.specInJson)
+    unresolvedRefs.forEach((pathStr, ref) => {
+      const err: any = {}
+      err.path = (pathStr as string[]).join(".")
+      err.message = `JSON Pointer points to missing location:${ref}`
```

Comment:
How about change the message to `The reference cannot be resolved`?

Reply:
this is detail message , there is a  summary description  the same as you mentioned